test/inductor/test_inductor_freezing.py

```diff
@@ -217,6 +217,10 @@ def foo(mod, inp):
                 ).run(code[0])
                 self.assertEqual(out_eager, out)
 
+    # freezing_discard_parameters is an unsound optimization, and it does not work with the attribute
+    # lifting strategy in dynamo. Dynamo needs this strategy to preserve correctness, which trumps speed
```

Comment:
it's not an unsound optimization, and we should find a better strategy than just disabling existing tests

Reply:
fine